proto/stride/stakeibc/tx.proto

```diff
@@ -314,3 +316,26 @@ message MsgUpdateTradeRoute {
   ];
 }
 message MsgUpdateTradeRouteResponse {}
+
+// Registers a community pool rebate by specifying the amount of the liquid
+// stake
+message MsgRegisterCommunityPoolRebate {
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "stride/x/stakeibc/MsgRegisterCommunityPoolRebate";
+
+  // Message signer
+  string creator = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+  // Chain ID of the community pool
```

Comment:
suggest: chain id of the chain whose community pool has a liquid staking rebate arrangement with stride

Reply:
done

Reply:
lgtm